src/humanloop/otel/exporter.py

```diff
@@ -0,0 +1,148 @@
+from queue import Queue
+from threading import Thread
+import typing
+from opentelemetry import trace
+from opentelemetry.sdk.trace import Span
+from opentelemetry.sdk.trace.export import SpanExporter, SpanExportResult
+
+from humanloop.otel.constants import HL_FILE_OT_KEY, HL_LOG_OT_KEY, HL_TRACE_METADATA_KEY, OT_EMPTY_ATTRIBUTE
+from humanloop.otel.helpers import read_from_opentelemetry_span
+
+if typing.TYPE_CHECKING:
+    from humanloop.base_client import BaseHumanloop
+
+
+class HumanloopSpanExporter(SpanExporter):
+    """SpanExporter that uploads OpenTelemetry spans to Humanloop Humanloop spans."""
+
+    WORK_THREADS = 8
```

Comment:
is this user defined? 8 feels like too high a default number.

In the doctring i expect to read details around how threading is working

Reply:
Added a constructor argument so the user can specify the number. If not provided, it will now default to 4